### PR TITLE
Small https fixups

### PR DESCRIPTION
**Pull request by patcon**
Reticketed from https://github.com/edgi-govdata-archiving/100days/issues/4#issuecomment-309471257

h/t @ebarry

### CODE REVIEW

**Comment by patcon** [comment]
WIP. only skimmed index, so just realized i should check other pages

**Comment by patcon** [comment]
ok, good for review now @shaqsingh 

I've uploaded a tarball to our hosting, and it just need to be decompressed to get these changes live. (I can do that when this is +1'd, but so can someone else if need be)

File: _config.yml [hunk 1]
```diff
@@ -13,12 +13,13 @@
...
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: 100 Days Report
+title: 100 Days and Counting &ndash; EDGI
 email: EnviroDGI@protonmail.com
 description: >
   EDGI is producing a series of reports on the early days of the Trump administration.  In these reports, EDGI authors systematically investigate topics including the historical precedents for Trump’s attack on the EPA, consequences for toxics regulation and environmental justice, the influence of the fossil fuel industry on the new administration, changes to the public presentation of climate science, and the new administration’s hostility to scientific research and evidence.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://100days.envirodatagov.org" # the base hostname & protocol for your site, e.g. http://example.com
+production_url: "https://100days.envirodatagov.org" # Used for canonical url meta tag
```
Comment (dcwalk) [inline _config.yml:22]: why not just use url above?

File: index.html [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
 	  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-	  <title>100 Days</title>
+    <title>{{ site.title }}</title>
     <meta name="description" content="{{ site.description }}">
 
 	  <link rel="stylesheet" href="/assets/main.css">
-	  <link rel="canonical" href="http://localhost:4000/">
-	  <link rel="alternate" type="application/rss+xml" title="100 days" href="/feed.xml">
+    <link rel="canonical" href="{{ site.production_url }}{{ page.url | replace:'index.html',''}}">
```
Comment (dcwalk) [inline index.html:15]: nice, tho again not sure if a separate production_url is needed! 

But eventually this probably for the template moreso than a static index?

**Review by dcwalk — COMMENTED** [review]
Looking good -- only thought on whether production_url is needed seperate from url?

licensebuttons looks neat :)

**Review by dcwalk — APPROVED** [review]

File: _config.yml (hunk 1 above)
Comment (patcon) [reply _config.yml:22]: When auto-generating locally, jekyll replaces `site.url` with `localhost:4000`, no matter what you set it to. Might be a way to force this with environment variables or something, but this struck me as less error-prone for ppl autogenerating the site locally for hosting on namecheap :)

File: index.html (hunk 2 above)
Comment (patcon) [reply index.html:15]: samesies

**Comment by patcon** [comment]
ok, done and now live at https://100days.envirodatagov.org/

Pls re-open the right issue if we missed one that was marked as closed :)

**Comment by ebarry** [comment]
looks amazing Go Team!

--

+1 336-269-1539 / @lizbarry <http://twitter.com/lizbarry> / lizbarry.net

**Pull request merged**